composer.json

```diff
@@ -27,12 +27,13 @@
     },
     "require": {
         "php": "^5.6 || ^7.0",
-        "composer-plugin-api": "^1.0",
+        "composer-plugin-api": "^1.0 || ^2.0",
         "laminas/laminas-component-installer": "^1.0 || ^2.1.2",
         "laminas/laminas-zendframework-bridge": "^1.0"
     },
     "require-dev": {
-        "composer/composer": "^1.4.1",
+        "composer/composer": "^1.4.1 || ^2.0@dev",
+        "composer/semver": "^1.5.1 || ^2.0@dev",
```

Comment:
@boesing we should now remove this, as we have already stable [semver 2.0](https://github.com/composer/semver/releases/tag/2.0.0).